backoffice/.envs/local/.django

```diff
@@ -19,5 +20,5 @@ OPENSEARCH_HOST=opensearch:9200
 OPENSEARCH_INDEX_PREFIX=backoffice-backend-local
 
 # Airflow
-AIRFLOW_BASE_URL=http://host.docker.internal:8082
-AIRFLOW_TOKEN=CHANGE_ME
+AIRFLOW_BASE_URL=http://host.docker.internal:8080
+AIRFLOW_TOKEN=YWlyZmxvdzphaXJmbG93Cg
```

Comment:
is it okay to have tokens explicitly defined?

Reply:
Also is this token correct? it seems to have an addition `Cg` in the end if we are simply encoding `user:password`

Reply:
These are all local tokens, that we define in the fixtures for easier use, we don't use them anywhere